### DISCUSSION TITLE
我想下载私有GitHub仓库的release包，在package的add_urls中，我无法添加token认证，应该怎么做？

### DISCUSSION THREAD

**Discussion by leihuhu** [xmake-io/xmake#3269] [Q&A]
code

    if is_plat("windows") then
        
        add_urls("https://github.com/user/private_repo/releases/download/$(version)/xx.zip")

        add_versions("version", "f64d888c19123ee813ca82b3796857d68db09c3245d205b75167fe2572622884")

**Comment by xq114** [comment]
可以先下载到本地，然后用集成本地包的方法引入，url写本地地址
